odoo/addons/base/models/assetsbundle.py

```diff
@@ -22,11 +22,15 @@
 from odoo.modules.module import get_resource_path
 from .qweb import escape
 import psycopg2
-from odoo.tools import func, misc
+from odoo.tools import config, func, misc
+
+from odoo.addons.base.transpiler.transpiler_js import TranspilerJS
+from odoo.addons.base.transpiler.sourcemap_generator import SourceMapGenerator
 
 import logging
 _logger = logging.getLogger(__name__)
 
+last_version = {}
```

Comment:
definitely need a comment here. Also, name does not seem very descriptive. Maybe something like `bundle_checksum` would be clearer

Reply:
also, don't forget that odoo can be run with multiple workers.  In this case, it works, but it is worth keeping that in mind whenever doing that kind of stuff